modules/install/pages/migration.adoc

```diff
@@ -1,80 +1,99 @@
-* Scenario 2 - In case of bidirectional replication, for example, for data locality where Cluster 1 (Bucket A) has both ways replication streams to Cluster 2 (Bucket A’).
+== Migration during Unidirectional Replication
 
-For Scenario 1, perform the following steps to migrate your data to a new cluster:
+Follow these steps to migrate your data to a new bucket during unidirectional replication:
 
 . Stop application traffic coming into Cluster 1 (Bucket A).
 +
-NOTE: Allow __enough tim__e for the replication queues to drain.
-Check the *outbound XDCR mutation* statistics to confirm.
-For instructions, see xref:manage:monitor/ui-monitoring-statistics.adoc#outgoing_xdcr_stats[Monitoring Outgoing XDCR].
+NOTE: Allow enough time for the replication queues to drain.
+Confirm by reviewing the XDCR Total Outbound Mutations statistic derived from `xdcr_changes_left_total`.
+For more information, see xref:manage:monitor/monitor-intro.html#monitoring-with-the-ui[].
 
-. Stop and delete Replication stream to Cluster 2 (Bucket A’).
+. Stop and delete the replication stream to Cluster 2 (Bucket A’).
 For instructions, see
 xref:manage:manage-xdcr/xdcr-management-overview.adoc[XDCR Management Overview].
-. Run the `cbbackupmgr` tool to back up the entire bucket’s (Bucket A) data.
-For instructions, see  xref:backup-restore:enterprise-backup-restore.adoc[Backup].
+
+. Run the `cbbackupmgr` tool to back up the bucket’s (Bucket A) data.
+For instructions, see xref:backup-restore:enterprise-backup-restore.adoc[Backup].
+
 . Delete Bucket A from Cluster 1.
 For instructions, see
 xref:manage:manage-buckets/delete-bucket.adoc[Delete a Bucket].
-. Upgrade Cluster 1 to Couchbase Server version 4.6.0.
-For instructions, see xref:upgrade.adoc[Upgrading Couchbase Server].
-. Create a new Bucket B with the conflict resolution type as *Timestamp* selected.
+
+. Create Bucket B on Cluster 1 and select the Timestamp-based Conflict Resolution type.
 For instructions, see
 xref:manage:manage-buckets/create-bucket.adoc[Create a Bucket].
-. Run the `cbbackupmgr` tool to restore data.
-When restoring data from backup (use the [.cmd]`--force-updates` option).
-Make sure to disable the *Conflict Resolution* option during the restore.
-This is required since the conflict resolution types of the source and destination clusters do not match.
-. Once the restore operation is completed on Cluster 1, delete Bucket A’ from Cluster 2.
-. Upgrade Cluster 2 to Couchbase Server version 4.6.0.
-. Create a new Bucket B’ with the Conflict Resolution type as *Timestamp* selected.
+
+. Run the `cbbackupmgr` tool on Cluster 1 to restore data to Bucket B.
+Use the xref:backup-restore/cbbackupmgr-restore.adoc[`cbbackupmgr restore`] command with the option `--force-updates` to restore data from the backup.
+This option disables conflict resolution during the restore,
+and it's needed because the bucket you're restoring to has a different conflict resolution type than the bucket the backup is from.
```

Comment:
```suggestion
and it's needed because the bucket you're restoring to has a different conflict resolution type than the bucket the backup originated from.
```